openapi/kas-fleet-manager.yaml

```diff
@@ -1032,14 +1032,6 @@ paths:
           schema:
             type: boolean
           required: true
-        - in: query
```

Comment:
What is the reason this query parameter removed?

Reply:
Revised behaviour of the deregistration endpoint:

Previously: don't delete cluster if there are Kafka instances on it unless `force` is passed.
Now: never delete a cluster when there are kafka instnaces present